.github/workflows/pulsar-ci.yaml

```
@@ -465,6 +465,12 @@ jobs:
     timeout-minutes: 60
     needs: ['preconditions', 'build-and-license-check']
     if: ${{ needs.preconditions.outputs.docs_only != 'true'}}
+    strategy:
+      fail-fast: false
+      matrix:
+        platforms:
```

```suggestion
        platform:
```

It's better to not use a plural for matrix parameters.